app/resources/forms.py

```diff
@@ -40,3 +41,14 @@ class ResourceForm(Form):
         Length(5, 5)
     ])
     submit = SubmitField('Add Resource')
+
+
+class ReviewForm(Form):
+    rating = IntegerField('Rating', validators=[
```

Comment:
also, would this be better as a select dropdown (so people know what they are supposed to input)